Talentas/superset
# Extending Row Level Security

Locations Extended Row Level Security and Dev Server Running Docker Compose.
Example:
feature(RI-91): extending the Row Level Security for User Locations.
feature(RI-8): extending the authentication with okta provider. 

### SUMMARY
We are giving interface for adding Row Level Security for Locations and Meta database for Superset Captures and Uses in Custom Jinja Template to add where condition in 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

### TESTING INSTRUCTIONS
- Create User Role and Assign user StoreManger.
- Assign Store Manager User to Locations in User Data Access 
 - Check the Store MangerUser Logging for Dashboard/Chart he has access to see data.

### ADDITIONAL INFORMATION
  - [] Confirm DB migration upgrade and downgrade tested
